deployment/template-cronjob.yaml

```diff
@@ -189,3 +191,9 @@ parameters:
   name: CRON_SCHEDULE
   # Run every monday 1.am
   value: "0 1 * * MON"
+
+- description: "Max character limit that we are allowing for string such as 'body' while saving data in csv file."
+  displayName: String character Limit
+  required: true
+  name: STRING_CHAR_LIMIT
```

Comment:
Can you make it more specific? like MAX_BODY_LENGTH?

Reply:
Kept it generic so other place also if we want to use this restriction so we can use the same parameter.